test/builder/builder.go

```diff
@@ -171,6 +175,10 @@ func GetTaskRunListWithName(c *framework.Clients, tname string) *v1alpha1.TaskRu
 		log.Fatalf("Couldn't get expected taskRunList  %s", err)
 	}
 
+	if err == nil && sortByStartTime {
```

Comment:
did you mix some patch this addition doesn't seem to belong to this pr or have i missed something ?

Reply:
Yes, see #1115. Currently our e2e tests are not always using the correct PipelineRun or TaskRun due to them not being returned ordered by start time. Added to this pr as it was affecting it.